services/frontend/src/components/Modal/network/IPam.tsx

```diff
@@ -120,11 +122,7 @@ const IPam = () => {
         renderField={(
           element: ReactElement,
           field: IFieldType
-        ): ReactElement => (
-          <Field style={{ paddingLeft: field.type === "records" ? 16 : 0 }}>
-            {element}
-          </Field>
-        )}
```

Comment:
Why did you remove this? It adds padding to records within records.

Reply:
Its not necessary with new the design I am slowly rolling out.